lib/cell.ex

```diff
@@ -0,0 +1,18 @@
+defmodule GameOfLife.Cell do
+  def live(cell_map, location) do 
+      %{cell_map | location => :alive}  
```

Comment:
Try to be consistent with naming. If you call this "cell_map", you are implying the type, which is not the best way to name things in case they change. I think "cell" is fine here.